src/python/py/models/README.md

```diff
@@ -158,6 +158,20 @@ python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o p
 python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p precision -e execution_provider -c cache_dir_to_store_temp_files --extra_options enable_cuda_graph=1
 ```
 
+#### Quantization Provider
```

Comment:
Can you add a link to this section in the table of contents after "Enable Cuda Graph"?

https://github.com/microsoft/onnxruntime-genai/blob/cfb37d495db9067a0ea1402aa7ff2f76657830de/src/python/py/models/README.md?plain=1#L5-L22